docs/content/configuration/authentication.md

```diff
@@ -43,8 +43,8 @@ weight: 1
     $env:ATHENS_STORAGE = "$(Join-Path $pwd athens-storage)"
     $env:ATHENS_SVN = "$(Join-Path $pwd .subversion)"
     md -Path $env:ATHENS_STORAGE
-    docker run -d -v "$($env:ATHENS-STORAGE):/var/lib/athens" `
-      -v "$($env:ATHENS-SVN):/root/.subversion" `
+    docker run -d -v "$($env:ATHENS_STORAGE):/var/lib/athens" `
+      -v "$($env:ATHENS_SVN):/root/.subversion" `
```

Comment:
Is this on the host? If it is on the host, this could be any envirnoment variable for ATHENS-SVN right?

Reply:
Environment variables can only have alphanumeric characters and underscores as far as I remember.

Anyways, the problem is that there is a mismatch with the variable definition (in this case line 44) and it's usage (in this case line 47)